<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardConfigWidget</class>
 <widget class="QWidget" name="CardConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,8,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>行数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRow">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>列数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxCol">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetLayout">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRecognise">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>识别</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ScreenToolBar" name="screenBar" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MontageLayout" name="wx_layout" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MontageLayout</class>
   <extends>QWidget</extends>
   <header>montagelayout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenToolBar</class>
   <extends>QWidget</extends>
   <header>screentoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
